engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/tasks.module.js

```diff
@@ -27,23 +27,10 @@ angular.module('Bastion.tasks').config(['$stateProvider', function ($stateProvid
         abstract: true,
         templateUrl: 'tasks/views/tasks.html'
     })
-    .state('tasks.index', {
-        url: 'katello_tasks',
-        permission: 'view_tasks',
-        templateUrl: 'tasks/views/tasks-index.html'
-    })
-    .state('tasks.details', {
```

Comment:
I don't think these are used, searched with something like `git grep \'tasks.index\' engines/`

Reply:
Try `git grep \"tasks.details\" engines/` to find [it](https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/views/tasks-index.html#L1).

Reply:
Thanks - I restored the tasks.details state. I didn't investigate but I think it's possible the view referencing it is not actually used. Maybe some tech debt, but not going to clean it all up here.